### PR TITLE
More about tools!

### PR DESCRIPTION
**Pull request by robinandeer**
I've added some more bit and pieces about git and python, namely:

- grip for previewing markdown
- an intro to logging in python
- that we prefer Python 3 for all code

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Python
 
-[Testing](testing.md)
+Python is a popular and easy-to-learn interpreted programming language. To get a feel for the language, take a look at this [comprahensive reference][python-ref]. With the surge in adoption of the most recent version of Python we are incouraging all new code to support [Python 3][python3] _first and foremost_.
```
Comment (ingkebil) [inline python/README.md]: *comprehensive

**Comment by ingkebil** [comment]
Looking good! :+1:

File: python/README.md (hunk 1 above)
Comment (robinandeer) [inline python/README.md]: yes! and "encouraging" 😜

**Comment by robinandeer** [comment]
I made that whole commit _inside_ of my editor ^^

**Pull request merged**